### ISSUE TITLE
Review and improve usage of target="_blank"

### ISSUE THREAD

**Issue by afercia** [Yoast/yoast-components#370]
Noticed a few places where we're using `target="_blank"` and the message to inform users the link is going to open a new browser tab/window is missing or hardcoded:

1
Onboarding WIzard failure message: to make this message appear, just set "Offline" in the Chrome console network panel and click on the "Next" button:

![screen shot 2017-11-06 at 12 15 41](https://user-images.githubusercontent.com/1682452/32439474-34ef9fa0-c2ef-11e7-9578-ce4f085a0568.png)

in this case, the message is missing.

2
Dashboard widget:

![screen shot 2017-11-06 at 12 28 18](https://user-images.githubusercontent.com/1682452/32439489-4ace94d4-c2ef-11e7-85bf-5103b5eaf73e.png)

- on the first 2 links, there's a hardcoded string `( Opens in a new browser tab )` which is not translatable
- on the link in the footer, the message is missing 

Worth noting WordPress has started using icons to give also a visual indication of links with `target="_blank"`. See for example the "nearby events" widget. This is not a standard yet across WordPress, but maybe worth considering to use it.

![screen shot 2017-11-06 at 12 27 29](https://user-images.githubusercontent.com/1682452/32439576-b30c0dc4-c2ef-11e7-92b1-4e9e56562c50.png)

**Comment by afercia** [comment]
There's now also the Cornerstone checkbox, its label has a link with `target="_blank"`. See #439 and https://github.com/Yoast/wordpress-seo/issues/7288
